<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Альбом</string>
  </property>
  <widget class="QLineEdit" name="cover_line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cover_button">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QLabel" name="cover_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Обложка альбома:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="artist_line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="artist_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Исполнитель альбома:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="album_line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="album_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Наименование альбома:</string>
   </property>
  </widget>
  <widget class="QLabel" name="genre_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Жанр альбома:</string>
   </property>
  </widget>
  <widget class="QLabel" name="year_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Год выпуска альбома:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="year_line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="result_button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Поехали...</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="genre_line">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
